almdrlib/session.py

```diff
@@ -127,12 +127,14 @@ def _init_session(self, *args, **kwargs):
         else:
             self._access_key_id, self._secret_key = self._config.get_auth()
 
-        logger.debug("Initialized session. "
-                     f"access_key_id={self._access_key_id}, "
-                     f"account_id={self._account_id}, "
-                     f"profile={profile}, "
-                     f"global_endpoint={self._global_endpoint}"
-                     )
+        logger.info(
```

Comment:
afm too verbose for info log. 
profile is either default or --profile <name it>, so it known. Access key is derivative of profile really, why flashing it in the log.

Reply:
Changed it back to debug.
I would still love to see all of this printed since this there are multiple ways to initialize session.